src/app/models/body-obj.model.ts

```diff
@@ -0,0 +1,21 @@
+export class RegistrationBody {
+  username: string;
```

Comment:
Is this body for request or response? because I think the username will be available only in the response, because the API will generate it.

Reply:
A TS Class can also be used as an interface so I believe these definitions could be used for either.